app/Cargo.toml

```diff
@@ -18,6 +18,9 @@ hecs = { version = "0.9.0", default-features = false }
 rand = { version = "0.8.4", features = ["small_rng"], default-features = false }
 num = { version = "0.4.0", default-features = false}
 wavefront = { version = "0.2.3", default-features = false}
+hashbrown = "0.12"
+lazy_static = { version = "1.4.0", default-features = false, features = ["spin_no_std"] }
+micromath = { version = "1.0.1" }
```

Comment:
How about making `modulator`  a local crate, and putting these in a separate `Cargo.toml`?
I'm also thinking of putting GRRLIB in a separate local crate in the future, so it would be consistent of us.

Reply:
Good idea!

Reply:
This has now been done. Please re-review and hopefully merge 😊